Packs/BmcHelixRemedyForce/Integrations/BmcHelixRemedyForce/README.md

```diff
@@ -0,0 +1,953 @@
+BMC Helix Remedyforce integration allows customers to create/update service requests and incidents. It also allows to update status, resolve service requests and incidents with customer notes. This integration exposes standard ticketing capabilities that can be utilized as part of automation & orchestration.
+This integration was integrated and tested with version xx of BMCHelixRemedyforce
```

Comment:
Please add the appropriate version of BMC Remedy force.

Reply:
We can mention Version 2020 winter, which we used entirely for DEV/QA cycles.
When we gave you demo instance, they released 2020 summer now, but we checked commands are working as expected.
Also we asked to @Arsenikr  and team to provide 2019 version, as it does not give any option to us when we setup trial instance. Trial instance by default takes latest BMC version. We did not receive 2019 BMC version for testing.

Reply:
so you can mention 2020 winter or 2020 summer, as you prefer.

Reply:
ok we will keep winter version, with exact version number too

Reply:
We have added the version of **202002.79** in the README file.